### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Miking98**
Update readme with instructions on how to go from OMOP => MEDS => FEMR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,16 +18,49 @@ We recommend users start with our [tutorial folder](https://github.com/som-shahl
...
 
+## OMOP Data
+
+If you have OMOP CDM formated data, follow these instructions:
+
+1. Download your OMOP dataset to `[PATH_TO_SOURCE_OMOP]`.
+2. Convert OMOP => MEDS using the following:
+```bash
+# Install MEDS Python packages
+pip install meds meds-etl
```
Comment (EthanSteinberg) [inline README.md:42]: With the addition of meds_etl to the dependency list, I guess this isn't necessary anymore?

File: README.md [hunk 2]
```diff
@@ -18,16 +18,49 @@ We recommend users start with our [tutorial folder](https://github.com/som-shahl
...
+
+# Convert OMOP => MEDS
+meds_etl_omop [PATH_TO_SOURCE_OMOP] [PATH_TO_OUTPUT_MEDS]
```
Comment (EthanSteinberg) [inline README.md:45]: We are missing the stanford omop fixer call: https://github.com/som-shahlab/femr/blob/main/src/femr/post_etl_pipelines/stanford.py#L50

**Review by EthanSteinberg — CHANGES_REQUESTED** [review]
Yeah, we really need this additional documentation. Two minor fixes to it though.

**Comment by EthanSteinberg** [comment]
Shoot, accidentally broke this MR. Super sorry about that @Miking98. You should be able to recreate and reopen it.

**Comment by Miking98** [comment]
No worries, will reopen!

**Pull request closed without merging**